sql/analyzer/optimization_rules_test.go

```diff
@@ -218,7 +218,7 @@ func TestPushNotFilters(t *testing.T) {
 	ctx := sql.NewContext(context.Background(), sql.WithSession(sess))
 	ctx.SetCurrentDatabase("mydb")
 
-	b := planbuilder.New(ctx, cat, sql.NewMysqlParser())
+	b := planbuilder.New(ctx, cat, sql.NewMysqlParser(), planbuilder.DefaultAuthorizationHandler())
```

Comment:
not super important but i would have voted putting auth handler on catalog

Reply:
Agnostic on this point, either works